crates/backend/src/typegen.rs

```diff
@@ -354,19 +409,25 @@ pub fn ty_to_ts_type(ty: &Type, is_return_ty: bool, is_struct_field: bool) -> (S
             aliases
               .borrow()
               .get(rust_ty.as_str())
-              .map(|a| (a.to_owned(), false))
+              .map(|a| (a.to_owned(), false, false))
           });
-          ts_ty = type_alias.or(Some((rust_ty, false)));
+          ts_ty = type_alias.or(Some((rust_ty, false, false)));
         }
       }
 
-      ts_ty.unwrap_or_else(|| ("any".to_owned(), false))
+      ts_ty.unwrap_or_else(|| ("any".to_owned(), false, false))
     }
-    Type::Group(g) => ty_to_ts_type(&g.elem, is_return_ty, is_struct_field),
+    Type::Group(g) => ty_to_ts_type(&g.elem, is_return_ty, is_struct_field, false),
     Type::Array(a) => {
-      let (element_type, is_optional) = ty_to_ts_type(&a.elem, is_return_ty, is_struct_field);
-      (format!("{}[]", element_type), is_optional)
+      let (element_type, is_optional, _) =
+        ty_to_ts_type(&a.elem, is_return_ty, is_struct_field, false);
+      (format!("{}[]", element_type), is_optional, false)
+    }
+    Type::Paren(p) => {
```

Comment:
Add this to fix types of tuples with only one element that exists.

Reply:
great